tests.py

```diff
@@ -403,3 +404,24 @@ def test_load_config_files(self):
             assert yaml.load(config_file, Loader=yaml.FullLoader)
         with open("config_quick_debug.yml", "r") as config_quick_debug_file:
             assert yaml.load(config_quick_debug_file, Loader=yaml.FullLoader)
+
+
+class EndToEndTest:
+
+    def EndToEndTest(self):
```

Comment:
Not to forget this: would you kindly add a note about the required `test_` prefix in classes and methods to make a test discoverable by pytest, please?

Reply:
And please, no camel case in python! Ssssnake case rules here!